.travis/z3.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x
+
+# Make sure we exit if there is a failure
+set -e
+
+source ${KLEE_SRC}/.travis/sanitizer_flags.sh
+if [ "X${IS_SANITIZED_BUILD}" != "X0" ]; then
+  echo "Error: Requested Sanitized build but Z3 being used is not sanitized"
+  exit 1
+fi
+
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
```

Comment:
While we are here we could support specifying the Z3 version too. If you'd rather not tackle that now, that's fine.